pkg/reconciler/autoscaling/kpa/kpa.go

```diff
@@ -155,7 +155,8 @@ func (c *Reconciler) reconcile(ctx context.Context, pa *pav1alpha1.PodAutoscaler
 	// 1. The revision is scaled to 0.
 	// 2. The excess burst capacity is negative.
 	if want == 0 || decider.Status.ExcessBurstCapacity < 0 {
-		logger.Debugf("SKS %s is in proxy mode: want = %d, ebc = %d", pa.Name, want, decider.Status.ExcessBurstCapacity)
+		logger.Infof("SKS is in proxy mode: want = %d, ebc = %d", want, decider.Status.ExcessBurstCapacity)
+		logger.Debugf("SKS is in proxy mode: want = %d, ebc = %d", want, decider.Status.ExcessBurstCapacity)
```

Comment:
Do we need this message for both Debug and Info level? I wonder that we get duplicated message when setting debug level.